modules/microshift-etcd-version.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * microshift_troubleshooting/microshift-version.adoc
+// * microshift_support/microshift-etcd.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-version-etcd_{context}"]
+= Checking the etcd version
+
+To enhance troubleshooting, you can get the version information for the etcd included with your {microshift-short} version.
+
+.Procedure
+
+* Run the following command to check the version information:
++
+[source,terminal]
+----
+$ microshift-etcd version
+----
++
+.Example output
+[source,terminal,subs="attributes+"]
+----
+{microshift-short}-etcd Version: 4.15.0_0.nightly_arm64_2023_11_20_235936_20231129145639_153b6f993
```

Comment:
We should not use macros in executable names, please change to hardcoded `microshift-etcd`